lib/rules/no-typos.js

```diff
@@ -33,6 +33,7 @@ const LIFECYCLE_METHODS = [
 
 module.exports = {
   meta: {
+    type: 'suggestion',
```

Comment:
this feels more like a "problem"

Reply:
But `category: 'Stylistic Issues',`

Reply:
In the case of react components, i think that category is incorrect.